init_suite.py

```diff
@@ -11,6 +11,20 @@
 log = Log(__name__)
 
 
+def merge_dicts(dict1, dict2):
+    """Recursively merges dict2 into dict1"""
```

Comment:
please use google style docstrings
i.e., provide proper description about args used and return value